migrations/3_revenue_sharing.js

```diff
@@ -66,19 +55,15 @@ module.exports = async function (deployer, network) {
     const networkId = networkConfiguration.network_id;
 
     jsonConfig[networkId] = {
-        multisigOwner: multisigSafe,
-        relayOperator: relayOperator,
-        walletProvider: walletProvider,
-        liquidityProvider: liquidityProvider,
-        iovLabsRecipient: iovLabsRecipient,
-        relayOperatorShare: relayOperatorShare,
-        walletProviderShare: walletProviderShare,
-        liquidityProviderShare: liquidityProviderShare,
-        iovLabsRecipientShare: iovLabsRecipientShare,
         collectorContract: Collector.address,
-        collectorSafe: await collectorInstance.multisigOwner.call(),
+        collectorOwner: await collectorInstance.owner.call(),
         collectorToken: await collectorInstance.token.call(),
     };
 
+    revenueSharingPartners.forEach(function (partner, i) {
+        jsonConfig[networkId]["partner" + (i+1)] = partner.beneficiary;
```

Comment:
if possible, I would generate a JSON with a different structure for the partner parts:
```
{
   ...
   "partner1": {
        "address": "0x..."
        "share": "..."
   }
   ...
}
```

Reply:
much better! thanks, fixed